src/views/Login/index.tsx

```diff
@@ -27,12 +33,15 @@ const Login = () => {
   };
 
   const onSuccess = (tokens: any) => {
-    oktaAuth.handleLoginRedirect(tokens);
+    const referringUri = oktaAuth.getOriginalUri();
```

Comment:
This works well for routes that we draw even when the user is not logged in, but it doesn't work for situations, like the gov team, where we don't draw the routes ahead of time.

Reply:
In this case, we should be drawing routes for all users instead of conditionally rendering routes. I mentioned a potential fix for this in #856

Reply:
We're going to change how we draw the routes! In the meantime, here is the fix using the okta code.